src/main/java/org/matsim/episim/EpisimContainer.java

```diff
@@ -88,7 +88,13 @@
 	 * The id of the ReplayEventTask that handles the events for this container
 	 */
 	private int taskId = 0;
-	
+
+	/**
+	 * This counts the number of persons in this container
+	 * which have the DiseaseStatus contagious or showingSymptoms. 	
```

Comment:
Even though infected persons can not immediately infect others, they are important for contact tracing.
So I think the state `infectedButNotContagious` should also be included.

Reply:
I see a problem with this. If the contact tracing is dependent on the persons state, the results will not be deterministic anymore in the multi-threaded version.

Reply:
Never mind, infection events are not processed on the same day, so there should be no problem with determinism.